Themis/Views/Assessment/CorrectionSidebar/ProblemStatementCellView.swift

```diff
@@ -20,9 +20,35 @@ struct ProblemStatementCellView: View {
             Text("Problem Statement")
                 .font(.largeTitle)
 
+            ZStack {
+                Divider()
+                    .frame(height: 5)
+                    .overlay(.gray)
+                HStack {
+                    ForEach(vm.bundledTests, id: \.testCase) { bundledTest in
+                        if bundledTest != vm.bundledTests.first {
+                            Spacer()
+                        }
+                        if bundledTest.passed {
+                            Image("TestPassedSymbol")
+                                .resizable()
+                                .scaledToFit()
+                                .frame(maxWidth: 30, maxHeight: 30)
+                                .background(colorScheme == .light ? .white : .black)
+                        } else {
+                            Image("TestFailedSymbol")
+                                .resizable()
+                                .scaledToFit()
+                                .frame(maxWidth: 30, maxHeight: 30)
+                                .background(colorScheme == .light ? .white : .black)
+                        }
```

Comment:
Maybe this could be put into a separate file like `TestSymbol` or something like that? It seems like it could be easily moved for better reusability :-)

Reply:
I don't think this symbol is more often needed.